src/cache_cf.cc

```diff
@@ -961,7 +961,7 @@ configDoConfigure(void)
 #endif
 
     if (Security::ProxyOutgoingConfig.encryptTransport) {
-        debugs(3, DBG_IMPORTANT, "Initializing https:// proxy context");
+        debugs(3, Important(74), "Initializing https:// proxy context");
```

Comment:
FWIW, I have been thinking for a while that all these "Initializing ..." messages should be level 2+. Just have not gotten around to doing it. If you agree please do that instead.

Same goes for the section 83 "Using ..." messages shown on startup/reconfigure.

Reply:
Great, I will do that.